packages/sdk-client/src/client.js

```diff
@@ -71,6 +73,11 @@ export default function createClient (options: ClientOptions): Client {
       fn: ProcessFn,
       opt: ProcessOptions = { accumulate: true },
     ): Promise<Array<Object>> {
+      if (!fn)
+        throw new Error('Missing argument! Second argument must be defined')
```

Comment:
if the `function` is required, it's presence cannot be validated in the `validate` function. How would you suggest I do this?